pkg/client/ocm/types_test.go

```diff
@@ -0,0 +1,133 @@
+package ocm
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func Test_GetProduct(t *testing.T) {
+	type fields struct {
+		t KafkaQuotaType
+	}
+
+	tests := []struct {
+		name   string
+		fields fields
+		want   string
+	}{
+		{
+			name: "should return RHOSAK for standard quota",
+			fields: fields{
+				t: StandardQuota,
+			},
+			want: string(RHOSAKProduct),
+		},
+		{
+			name: "should return RHOSAKTrial for developer quota",
+			fields: fields{
+				t: DeveloperQuota,
+			},
+			want: string(RHOSAKTrialProduct),
+		},
+	}
+
+	RegisterTestingT(t)
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			Expect(tt.fields.t.GetProduct()).To(Equal(tt.want))
```

Comment:
[optional]
You can declare `want` as `KafkaProduct`, so you can avoid the castings to string.
Then you just have to change the test to:
```suggestion
			Expect(tt.fields.t.GetProduct()).To(BeEquivalentTo(tt.want))
```